study.md

```diff
@@ -66,15 +69,15 @@ nearest_star = the_sun
 Will modifying `sol` affect `the_sun`? Why or why not?
 
 ```md
-<!-- your answer here -->
+No, because they are separate objects being constructed by the initiator function Star.
 ```
 
 ## Ruby Object Equality: Part II
 
 Will modifying `the_sun` affect `nearest_star`? Why or why not?
 
 ```md
-<!-- your answer here -->
+Yes, because nearest_star inherits all of its attributes from the_sun.
```

Comment:
Not because it inherits any attributes, but because both identifiers are pointing to the same object in memory.